snapcraft.yaml

```diff
@@ -30,3 +35,15 @@ parts:
       - e2fsprogs
       - fakeroot
       - dosfstools
+      - squashfs-tools
+      - coreutils
+      - util-linux
+      - sed
+    override-build: |
+      snapcraftctl build
+      # create a symlink /usr/bin/fakeroot -> /usr/bin/fakeroot-tcp
+      cd $SNAPCRAFT_PART_INSTALL/usr/bin/
+      ln -s fakeroot-tcp fakeroot
+      # Create a symbolic link to /usr/lib/<arch> where libfakeroot will live
+      cd $SNAPCRAFT_PART_INSTALL/usr/lib/
+      ln -s *-linux-gnu* lib-arch
```

Comment:
I'm thinking maybe we could check if instead of symlinking a `lib-arch` directory we could use `$SNAP_ARCH_TRIPLET` - if it works, of course. Since I remember some talk about this snap environment variable in the past, but not sure if it's actually implemented and if it would work when used in the `environment:` section.

Reply:
$SNAP_ARCH_TRIPLET is only set during the snapcraft build/stage/prime, and not at the runtime of the snap. I could change `ln -s *-linux-gnu* lib-arch` to `ln -s $SNAP_ARCH_TRIPLET lib-arch`, but it still requires the symbolic link. If I put $SNAP_ARCH_TRIPLET in the FAKEROOT_LIB environment variable it will evaluate to "".